02_Architecture/01_README.md

```diff
@@ -0,0 +1,53 @@
+# Architecture
+
+Before going beyond a basic hello world and implementing the first real parts of our kernel, there are some key concepts about how the CPU operates that we have to understand. What is an interrupt, and how do we handle it? What does it mean to mask them? What is the GDT and what is it's purpose?
+
+It's worth noting that we're going to focus exclusively on x86 here, and some concepts are specific to this platform (the GDT, for example), while some concepts are transferable across most platforms (like a higher half kernels). Some concepts, like interrupts and interrupt handlers, are partially transferable to other platforms.
```

Comment:
should be better specify also that is 64 bit version?

Reply:
yes, makes sense :)